### PR TITLE
Feature/separate functions

### PR DESCRIPTION
**Pull request by PetervDooren**
I wanted to use the fitter from  the web gui also during the regular free mode. but the robocup plugin contains several functions which are not really related.

In this PR I separated the services required from the fitter gui and the map updater functionality. 

Before merging: ensure that the plugin names and parameters in the worldmodel parameter files in the bringup repos are updated.

### CODE REVIEW

File: ed_robocup/src/gui_plugin.cpp [hunk 1]
```diff
@@ -349,16 +313,6 @@ bool RobocupPlugin::srvFitEntityInImage(ed_robocup_msgs::FitEntityInImage::Reque
...
-
-        EntityRepresentation2D repr2d = fitter_.GetOrCreateEntity2D(e);
-        if (!repr2d.shape_2d.empty())
-        {
-            geo::Transform2 pose_2d;
-            pose_2d.t = geo::Vec2(fitted_pose.t.x, fitted_pose.t.y);
-            pose_2d.R = geo::Mat2(fitted_pose.R.xx, fitted_pose.R.xy,
-                                  fitted_pose.R.yx, fitted_pose.R.yy);
-            map_filter_.setEntityPose(pose_2d, repr2d.shape_2d, map_filter_padding_);
-        }
```
Comment (PetervDooren) [inline ed_robocup/src/gui_plugin.cpp:361]: This is the only connection between map updater and fitter.
This part makes sure that the newly fitted entity is added to the (gmapping) map. 
I know that this currently breaks this functionality, but I think there are better ways of implementing this. Right now the map is only updated if the new entity comes from the gui fitter, but this should react to all changes in position of entities within the worldmodel.
One way to do that is to check the revision number of ed perhaps.

Comment (MatthijsBurgh) [reply ed_robocup/src/gui_plugin.cpp:361]: Why should I accept any changes that breaks stuff. If you already have another solution in mind, please implement that.

Comment (PetervDooren) [reply ed_robocup/src/gui_plugin.cpp:361]: Hence the draft, I wanted to discuss a potentiap solution first

Comment (MatthijsBurgh) [reply ed_robocup/src/gui_plugin.cpp:361]: Ok

**Pull request closed without merging**